install_config/web_console_customization.adoc

```diff
@@ -1183,41 +867,36 @@ many modifications to the web console can be configured using
 xref:../install_config/install/advanced_install.adoc#advanced-install-configuring-global-proxy[the following parameters], which are configurable in the inventory file:
 
 - xref:changing-the-logout-url[`openshift_master_logout_url`]
-- xref:loading-custom-scripts-and-stylesheets[`openshift_master_extension_scripts`]
-- xref:loading-custom-scripts-and-stylesheets[`openshift_master_extension_stylesheets`]
-- xref:serving-static-files[`openshift_master_extensions`]
+- xref:loading-custom-scripts-and-stylesheets[`openshift_web_console_extension_script_urls`]
+- xref:loading-custom-scripts-and-stylesheets[`openshift_web_console_extension_stylesheet_urls`]
 - xref:serving-static-files[`openshift_master_oauth_template`]
```

Comment:
The serving-static-files section was removed, so we should remove this link to

Reply:
Yeah, this link was not correct to begin with. It should link to customizing the login page.